site/.vitepress/config.ts

```diff
@@ -367,11 +364,8 @@ export default defineConfig({
             text: "Public Client",
             base: "/packages/aa-core/client",
             collapsed: true,
+            link: "/",
             items: [
-              {
-                text: "Overview of Public Client",
-                link: "/",
-              },
```

Comment:
let's put this back, per my comment re: no section header pages to make each page explicitly visible on sidebar